asmcli/components/ca/managed-cas.sh

```diff
@@ -75,6 +75,59 @@ x_enable_workload_certificate_on_membership() {
      -d "${ENABLEFEATURE}" "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate?update_mask=membership_specs"
 }
 
+x_wait_for_gke_hub_api_enablement() {
+  info "Wait 3 minutes for GKE HUB API enablement to propagate to the systems ..."
+  sleep 180
+}
+
+x_wait_for_enabling_workload_certificates() {
+  local GKEHUB_API; GKEHUB_API="$1"
+  local FLEET_ID; FLEET_ID="$2"
+
+  info "Waiting for the workload certificates enablement. This may take up to 20 minutes ..."
+  # Check the status of workload certificate feature
+  local ENABLED; ENABLED=0
+  for i in {1..2}; do
+    local OUT; OUT="$(curl -H "X-Goog-User-Project: ${FLEET_ID}" \
+      -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+      "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate" 2>/dev/null | \
```

Comment:
can we directly use alpha command: ```gcloud alpha container fleet workload-certificate update --all-memberships --enable ``` instead of the https curl recall

Reply:
It is equivalent to the curl call and does not require the installation of an alpha version of gcloud.